bulbs/videos/static/videos/js/series/main.browserify.js

```diff
@@ -0,0 +1,27 @@
+var SeriesMeta = require('./series-meta/series-meta');
+var SeriesVideoGrid = require('./series-video-grid/series-video-grid');
+var LatestEpisode = require('./series-video-latest-episode/series-video-latest-episode');
+var SeriesVideoList = require('./series-video-list/series-video-list');
+
+SeriesPage = function() {
```

Comment:
This should be a separate file included via `require`, not inlined in the `main.browserify.js`